src/lib.rs

```diff
@@ -21,11 +20,10 @@ pub mod directwrite;
 #[cfg(windows)]
 pub use directwrite::DirectWriteRasterizer as Rasterizer;
 
-// If target is macos, reexport everything from darwin.
 #[cfg(target_os = "macos")]
-mod darwin;
+pub mod darwin;
 #[cfg(target_os = "macos")]
-pub use darwin::*;
+pub use darwin::CoreTextRasterizer as Rasterizer;
```

Comment:
What's the reason for those changes?

Reply:
There were several warnings related to unused code and missing `pub` declarations in the darwin module, and these were partly masked by the fact that everything in darwin is re-exported, while the other backends only export their version of `Rasterizer`. Now that the `set_font_smoothing` function is gone, we can do the same thing with the darwin backend and only export its Rasterizer. No other interfaces should be needed. I renamed it to match how the other backends have unique names for their Rasterizers.